src/activities/jumbled_words/Jumbled_words.qml

```diff
@@ -0,0 +1,486 @@
+                    }
+                    first++
+                    var last = cursorPosition
+                    for(; last < text.length; last++) {
+                        if(text.charCodeAt(last) === 8233)
+                            break
+                    }
+                    var line = getText(first, last)
+                    remove(first, last)
+                    insert(first, '<' + tag + '>' + line + '</' + tag + '>')
+                    cursorPosition = initialPosition
+                }
+            }
+        }
+
+        Rectangle {
+            id: submit
+            anchors.fill: submitText
+            border.color: "#FFFFFFFF"
+            border.width: 2
+            height:submitText.height
+            color: "#000065"
+            opacity: 0.31
+            radius: 10
+
+            MouseArea {
+                id: mouseAreaSubmit
+                anchors.fill: parent
+                hoverEnabled: ApplicationInfo.isMobile ? false : true
+                onClicked: {
+                    if(Activity.checkAnswer())
+                    {
+                        edit.text=""
+                        hintText.text="Hint"
+                        hintImage.source=""
+                    }
+                    else
+                    {
+                        edit.text=""
+                        hintText.text="Hint"
+                    }
+                 }
+            }
+        }
+
+        GCText {
+            id: submitText
+            anchors.horizontalCenter: parent.horizontalCenter
+            anchors{
+                bottom: bar.top
+                margins: 10
+            }
+            opacity: 1.0
+            text:qsTr("Submit")
+            fontSize: mediumSize
+            font.bold: true
+            style: Text.Outline
+            styleColor: "#2a2a2a"
+           color: "white"
+        }
+        GCText {
+            id: text
+            anchors {
+                left: parent.left
+                right:parent.right
+                top: questionItem.top
+                bottom: flick.top
+                margins: 10
+              }
+        }
+
+        Score {
+            id: score
+            anchors.top: parent.top
+            anchors.topMargin: 10 * ApplicationInfo.ratio
+            anchors.left: parent.left
+            anchors.leftMargin: 10 * ApplicationInfo.ratio
+            anchors.bottom: undefined
+            anchors.right: undefined
+        }
+
+        Bar {
+            id: bar
+            content: BarEnumContent { value: help | home | level | reload | config}
+            onHelpClicked: {
+                displayDialog(dialogHelpLeftRight)
+            }
+            onHomeClicked: activity.home()
+            onReloadClicked: {
+               edit.text = ''
+               hintText.text = 'Hint';
+               hintImage.source="";
+            }
+            onPreviousLevelClicked: {
+                hintImage.source="";
```

Comment:
you can probably do this in the initLevel function instead of doing it in each button